MultiVersions/Forge/main/integrateddynamics/lang/zh_tw.json

```
@@ -0,0 +1,1873 @@
+    "gui.integrateddynamics.partoffset.offsets.max.howtoincrease": "用設備強化：偏移量來增加最大值",
+    "gui.integrateddynamics.partoffset.slot.message.noint": "無效的整數座標",
+    "gui.integrateddynamics.partoffset.slot.message.outofrange": "偏移量超過最大值",
+    "gui.integrateddynamics.partoffset.x": "X 偏移量",
+    "gui.integrateddynamics.partoffset.y": "Y 偏移量",
+    "gui.integrateddynamics.partoffset.z": "Z 偏移量",
+    "gui.integrateddynamics.partsettings.side": "目標方向",
+    "gui.integrateddynamics.delay.capacity": "容量",
+    "gui.integrateddynamics.part.properties": "Properties:",
+    "gui.integrateddynamics.logicprogrammer.filter": "過濾器",
+    "gui.integrateddynamics.rewards": "獎勵",
+    "gui.integrateddynamics.rewards.collect": "領取",
+    "gui.integrateddynamics.rewards.collected": "已領取",
+    "gui.integrateddynamics.rewards.collectFailure": "無法領取獎勵，也許是背包太滿了。",
+    "gui.integrateddynamics.input": "輸入：%s",
+    "gui.integrateddynamics.output": "輸出：%s",
+    "gui.integrateddynamics.mechanical_squeezer.fluidautoeject": "自動輸出液體",
+    "gui.integrateddynamics.mechanical_squeezer.togglefluidautoeject": "自動輸出液體：%s",
+    "gui.integrateddynamics.mechanical_squeezer.togglefluidautoeject.info": "將液體輸出到鄰近的容器",
+    "gui.integrateddynamics.input_short": "輸入",
+    "gui.integrateddynamics.output_short": "輸出",
+    "gui.integrateddynamics.recipe.strictnbt": "嚴格檢查 NBT",
+    "gui.integrateddynamics.recipe.tagvariants": "Item tag variants",
+    "gui.integrateddynamics.recipe.reusable": "重複使用",
+
+    "_comment": "Diagnostics",
+    "gui.integrateddynamics.diagnostics.title": "Integrated Dynamics - Network Diagnostics",
+    "gui.integrateddynamics.diagnostics.parts": "Parts",
+    "gui.integrateddynamics.diagnostics.observers": "Storage Observers",
+    "gui.integrateddynamics.diagnostics.table.network": "Network",
+    "gui.integrateddynamics.diagnostics.table.cables": "Cables",
+    "gui.integrateddynamics.diagnostics.table.part": "Part",
+    "gui.integrateddynamics.diagnostics.table.ticktime": "Ticktime (ms)",
+    "gui.integrateddynamics.diagnostics.table.dimension": "Dimension",
+    "gui.integrateddynamics.diagnostics.table.position": "Position",
+    "gui.integrateddynamics.diagnostics.table.side": "Side",
+    "gui.integrateddynamics.diagnostics.table.actions": "Actions",
+
+    "_comment": "Keybindings",
+    "key.categories.integrateddynamics": "整合動力",
+    "key.integrateddynamics.logic_programmer_focus_search": "選取邏輯程式設計器搜尋欄",
+    "key.integrateddynamics.logic_programmer_open_rename": "邏輯程式設計器內重新命名變數",
+
+    "_comment": "Sound names",
+    "subtitles.integrateddynamics.effect.page.flipsingle": "翻頁聲",
+    "subtitles.integrateddynamics.effect.page.flipmultiple": "翻頁聲",
+
+    "_comment": "Blocks",
+    "block.integrateddynamics.cable": "邏輯纜線",
+    "block.integrateddynamics.variablestore": "變數箱",
+    "block.integrateddynamics.energy_battery": "儲電箱",
+    "block.integrateddynamics.energy_battery.info": "和其他儲電箱合成能夠增加容量，Shift + 右鍵可以開啟自動供應模式。",
```

原文與譯文皆為按下**Shift**，但實際上模組是識別**潛行**，如此翻譯較能避免誤導。
```suggestion
    "block.integrateddynamics.energy_battery.info": "和其他儲電箱合成能夠增加容量，潛行 + 按下右鍵可以開啟自動供應模式。",
```